src/games/strategy/triplea/ui/mapdata/MapData.java

```diff
@@ -173,15 +168,15 @@ public MapData(final ResourceLoader loader) {
       m_namePlace = PointFileReaderWriter.readOneToOne(loader.getResourceAsStream(prefix + TERRITORY_NAME_PLACE_FILE));
       m_kamikazePlace = PointFileReaderWriter.readOneToOne(loader.getResourceAsStream(prefix + KAMIKAZE_FILE));
       m_mapProperties = new Properties();
-      loadDecorations();
-      loadTerritoryNames();
+      m_decorations = loadDecorations();
+      m_territoryNameImages = territoryNameImages();
```

Comment:
Agree, but I've been trying to bite off just one thing at a time per commit, rather than refactoring everything off the bat (as much fun as that would be). So simply just haven't gotten to that change yet - but there are a lot of things that need to change, and many of these variables need to be refactored away.